dolphin-dnn/src/main/java/edu/snu/dolphin/dnn/NeuralNetworkDriverParameters.java

```diff
@@ -210,13 +239,14 @@ public Configuration getDriverConfiguration() {
         .bindNamedParameter(Delimiter.class, delimiter)
         .bindNamedParameter(MaxIterations.class, String.valueOf(maxIterations))
         .bindNamedParameter(InputShape.class, inputShape)
+        .bindNamedParameter(LogPeriod.class, String.valueOf(logPeriod))
         .build();
   }
 
   /**
-   * @return {@code true} if this Neural Network application uses REEF Group Communication, {@code false} otherwise
+   * @return the {@link ProviderType} of the injected neural net configuration
```

Comment:
This comment is misleading since `providerType` is in a protobuf neural network configuration file that is not injected, but is loaded from local or distributed filesystems.